Editor/ProjectAuditor.cs

```diff
@@ -303,13 +326,20 @@ public static string GetCategoryName(IssueCategory category)
         /// Number of available built-in and registered categories
         /// </summary>
         /// <returns> Returns the number of available categories</returns>
-        public static int NumCategories()
+        internal static int NumCategories()
         {
             return (int)IssueCategory.FirstCustomCategory + s_CustomCategories.Count;
         }
 
+        /// <summary>
+        ///   <para>Returns the relative callback order for callbacks.  Callbacks with lower values are called before ones with higher values.</para>
```

Comment:
Tiny detail:  Usually we didn't add <para> to the summaries. Probably may be removed while keeping the same layout!?